package/scripts/ambari_helpers.py

```diff
@@ -44,3 +44,10 @@ def cdap_config(name=None):
   )
 
   Execute('update-alternatives --install /etc/cdap/conf cdap-conf ' + params.cdap_conf_dir + ' 50')
+
+def has_hive():
```

Comment:
It could be. Or it could be called "explore_supported" or whatever, since it's really a check that we have the requisite service.

However, this made me realize that I am missing a part of the check that is necessary. Mainly, that the host with CDAP Master on it is also a Hive Client. I'll do that in another PR.